tests/integration-tests/tests/schedulers/test_slurm/test_fast_capacity_failover/pcluster.config.yaml

```diff
@@ -9,7 +9,11 @@ HeadNode:
 Scheduling:
   Scheduler: slurm
   SlurmQueues:
-    - Name: queue1
+{% for strategy in scaling_strategies %}
+    - Name: queue-{{ strategy }}
+      {% if strategy == "job-level" %}
+      JobExclusiveAllocation: true
+      {% endif %}
```

Comment:
Nitpick: how many strategies are there? Because a jinja loop can be convenient but impacts the readability of the template. If we are generating just a couple of queues, I would write the template explicitly rather than this way.